kytos/core/api_server.py

```diff
@@ -147,12 +147,10 @@ def get_ui_components(self, filename):
         components = []
         for name in glob(path):
             group = name.split('/')
```

Comment:
I suggest making it clear in the code what is documented only on the commit message, as a comment or on the variables' names:
username, napp_name, component_section, filename = group.split('/')

Reply:
@macartur please do a double check if this suggestion is possible.

Reply:
Fixed, I changed from `group` to `dirname`.